content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/create.md

```diff
@@ -0,0 +1,49 @@
+---
+title: influxctl cluster create
```

Comment:
The only cluster subcommands that should be visible via `influxctl cluster -h` by default are `list` and `get`. Other commands may have help output, but a user won't know about them unless they guess